verification/verifier.go

```diff
@@ -124,30 +138,154 @@ func (v *Verifier) Verify(ctx context.Context, artifactUri string) ([]*Signature
 }
 
 func (v *Verifier) processSignature(ctx context.Context, sigBlob []byte, sigManifest registry.SignatureManifest, trustPolicy *TrustPolicy, outcome *SignatureVerificationOutcome) error {
```

Comment:
The caller method `processSignature` calls `processSignature` even after it encounters a valid signature, at line 113. Should we not stop processing signature at that point?